wallet/src/key_chain/mod.rs

```diff
@@ -99,6 +99,8 @@ pub enum KeyChainError {
     NoPrivateKeyFound,
     #[error("No VRF private key found")]
     NoVRFPrivateKeyFound,
+    #[error("No standalone address found")]
+    NoStadaloneAddressFound,
```

Comment:
Standalone

Reply:
Same here in these errors... we can include more information in all these errors.